huitparfait-auth/src/user-service.js

```diff
@@ -1,9 +1,13 @@
-import Bluebird from 'bluebird'
+import Config from './infra/config'
 import { sign } from './infra/jwt'
+import request from 'request-promise'
+
+const apiClient = request.defaults({ baseUrl: Config.get('proxy.url') })
```

Comment:
ajouter `simple:true`

=> simple which is a boolean to set whether status codes other than 2xx should also reject the promise
https://github.com/request/request-promise#api-in-detail